.changes/remove-webview-ivar.md

```diff
@@ -0,0 +1,5 @@
+---
+"wry": patch
+---
+
+Remove redundant webview ivar in WryWebView.
```

Comment:
I think this should instead explain the issue to solve rather than what the actual fix is.
```suggestion
On iOS, fix panic at runtime due to setting webview ivar.
```